packages/cli/src/commands/server/eventsSocket.ts

```diff
@@ -0,0 +1,161 @@
+/**
+ * (c) Facebook, Inc. and its affiliates. Confidential and proprietary.
+ *
+ * @format
+ */
+
+import {Server as WebSocketServer} from 'ws';
+import {logger} from '@react-native-community/cli-tools';
+import prettyFormat from 'pretty-format';
+import {Server as HttpServer} from 'http';
+import {Server as HttpsServer} from 'https';
+import messageSocketModule from './messageSocket';
+
+type Server = HttpServer | HttpsServer;
+
+type Command = {
+  version: number;
+  type: 'command';
+  command: string;
+  params?: any;
+};
+
+const PROTOCOL_VERSION = 2;
```

Comment:
What is a `PROTOCOL_VERSION`? Something Flipper-related? Would be great to have a comment.